### DISCUSSION TITLE
Issue with setting up Web3 client with Matic

### DISCUSSION THREAD

**Discussion by vjumminz** [0xPolygon/matic.js#300] [Q&A] [answered]
I have just begun working with matic.js and am having difficulty setting up the Web3ClientPlugin. I am getting a type error when trying to use the use function: 
```
return pluginInstance.setup.apply(pluginInstance, __spreadArrays([_default__WEBPACK_IMPORTED_MODULE_0__["defaultExport"]], payload));
                                ^

TypeError: Cannot read property 'apply' of undefined
```
My code:
```
const {use, POSClient} = require("@maticnetwork/maticjs");
const Web3ClientPlugin = require('@maticnetwork/maticjs-web3')

use(Web3ClientPlugin)
```

My package.json:
```
{
  "name": "tests",
  "version": "1.0.0",
  "description": "",
  "main": "test.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@maticnetwork/maticjs": "^3.0.0",
    "@maticnetwork/maticjs-web3": "^1.0.0"
  }
}
```
Have spent a couple hours trying to figure this out, even tried using different versions of matic.js. Getting the same issue with ethers as well. Assuming it's something simple I'm missing but I've stripped my code down to the most basic implementation of matic.js and still can't find the issue. Any help would be appreciated.

**Comment by ujjwalguptaofficial** [comment, marked as answer]
your code is - 

```
const Web3ClientPlugin = require('@maticnetwork/maticjs-web3')
```

change it to - 

```
const { Web3ClientPlugin } = require("@maticnetwork/maticjs-web3");
```

take a look at our example code - https://github.com/maticnetwork/matic.js/blob/master/examples/utils.js

**Comment by vjumminz** [comment]
Such a simple fix. Appreciate the help!
